elixir/apps/domain/lib/domain/mailer/auth_email.ex

```diff
@@ -1,18 +1,18 @@
-defmodule Web.Mailer.AuthEmail do
-  use Web, :html
+defmodule Domain.Mailer.AuthEmail do
   import Swoosh.Email
-  import Web.Mailer
+  import Domain.Mailer
+  import Phoenix.Template, only: [embed_templates: 2]
 
-  embed_templates "auth_email/*.html", suffix: "_html"
-  embed_templates "auth_email/*.text", suffix: "_text"
+  embed_templates("auth_email/*.html", suffix: "_html")
+  embed_templates("auth_email/*.text", suffix: "_text")
```

Comment:
Formatter here should not make this change. You need to move some rules from `web`'s `.formatter.exs` to `domain`.